config/routes.php

```diff
@@ -24,4 +24,9 @@
 $route->get('/stats', 'Metrics\\Controller@stats');
 
 // API
+$route->get('/api/angeltypes/my', 'ApiController@getMyAngelTypes');
```

Comment:
`$route->addGroup('/api', function (RouteCollector $route) { [...]` should be used here

Reply:
Also as mentioned above: Atm the API can't be implemented in a stable way so it should use a version namespace (normally `/api/v[\d+]/`), so in that case it should be `/api/01-alpha` or something as it will change unexpectedly.